WcaOnRails/app/models/team_member.rb

```diff
@@ -12,7 +12,7 @@ def current_member?
 
   validate :start_date_must_be_earlier_than_end_date
   def start_date_must_be_earlier_than_end_date
-    if end_date != nil && start_date >= end_date
+    if start_date && end_date && start_date >= end_date
```

Comment:
Does this change mean we're ok with either leaving the start date empty or leaving the end date empty? I think I'd rather not support empty start date, and if we do decide to support that, we may need to update other code to be aware of that.